index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] [OPTIONAL] Good job adding comment 😃 , however, it could be better if you add emojis to make your comment more friendly 👍